stable-diffusion/README.md

```diff
@@ -0,0 +1,69 @@
+
+
+<h2><p align="center">Stable Diffusion Finetuning</p></h2>
+
+<h3><p align='center'>
+<a href="https://www.mosaicml.com">[Website]</a>
+- <a href="https://docs.mosaicml.com/">[Composer Docs]</a>
+- <a href="https://docs.mosaicml.com/en/stable/method_cards/methods_overview.html">[Methods]</a>
+- <a href="https://www.mosaicml.com/team">[We're Hiring!]</a>
+</p></h3>
+
+<p align="center">
+    <a href="https://join.slack.com/t/mosaicml-community/shared_invite/zt-w0tiddn9-WGTlRpfjcO9J5jyrMub1dg">
+        <img alt="Chat @ Slack" src="https://img.shields.io/badge/slack-chat-2eb67d.svg?logo=slack">
+    </a>
+    <a href="https://github.com/mosaicml/examples/blob/main/LICENSE">
+        <img alt="License" src="https://img.shields.io/badge/License-Apache%202.0-green.svg?logo=slack">
+    </a>
+</p>
+<br />
+
+This folder contains starter code for finetuning Stable Diffusion on the lambda labs pokemon dataset. You can finetune Stable Diffusion v1 or 2, save the model and generate your own pokemon, or swap out the dataset to adapt stable diffusion to your desired domain.
+
+
+# Quick Start
+
+## Clone the repo
+```bash
+git clone mosaicml/examples
+cd examples/stable-diffusion
+```
+## Install requirements
+
+```
+pip install -r requirements.txt 
+```
+## Train the model
```

Comment:
Could you provide some details on what will be trained? E.g. maybe stable diffusionv1.4 for 2000 batches on pokemon? I'm not certain what to add, but I imagine it would be nice to know what to expect from running this command